lib/ansible/modules/find.py

```diff
@@ -154,6 +154,18 @@
             - When doing a C(contains) search, determine the encoding of the files to be searched.
         type: str
         version_added: "2.17"
+    checksum_algorithm:
+        description:
+        - Algorithm to determine checksum of file.
+        - Will throw an error if the host is unable to use specified algorithm.
+        - The remote host has to support the hashing method specified, V(md5)
+            can be unavailable if the host is FIPS-140 compliant.
+        - Availability might be restricted by the target system, for example FIPS systems won't allow md5 use
```

Comment:
redundant phrases, also update get_checksum description

Reply:
also might go into a docfragment and share with stat module

Reply:
May I add my name after the Copyright tag in the docfragment file, or should I just keep it as 'Ansible Project'?